googletest/test/gtest_catch_exceptions_test_.cc

```diff
@@ -138,7 +138,7 @@ TEST_F(CxxExceptionInConstructorTest, ThrowsExceptionInConstructor) {
 }
 
 // Exceptions in destructors are not supported in C++11.
-#if !defined(__GXX_EXPERIMENTAL_CXX0X__) &&  __cplusplus < 201103L
+#if !GTEST_LANG_CXX11
```

Comment:
We have warning here on VS2015

    ~CxxExceptionInDestructorTest() {
       GTEST_SUPPRESS_UNREACHABLE_CODE_WARNING_BELOW_(
           throw std::runtime_error("Standard C++ exception"));
    }